README.md

```diff
@@ -43,12 +42,31 @@ fastify.listen(3000, function (err) {
 })
 ```
 
+with async-await:
+
+```js
+// Require the framework and instantiate it
+const fastify = require('fastify')()
+
+// Declare a route
+fastify.get('/', async (request, reply) => {
+  reply.type('application/json').code(200)
```

Comment:
Actually this is not needed, since `application/json` and `200` are our defaults :D